docs/models/elastigroup/azure_v3.md

```diff
@@ -395,13 +395,13 @@ __Arguments__
 ```python
 Secret(self,
        source_vault='d3043820717d74d9a17694c176d39733',
-       vault_certificate='d3043820717d74d9a17694c176d39733')
+       vault_certificates='d3043820717d74d9a17694c176d39733')
```

Comment:
is that secrets ok?, should it be a placeholder?

Reply:
This is not the actual secret. It is the random value that is assigned to the "none" so that partial update can be performed. Only change performed now is to change vault_certificate to vault_certificates so that it is consistent with the specification.
![image](https://user-images.githubusercontent.com/86176942/220548665-e09399d2-8402-4fbb-9de3-6a7291425c8f.png)